templates/about-base.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block header %}
+<div class="cratesfyi-package-container">
+<div class="container about">
+	<h3 id="crate-title">Docs.rs documentation</h3>
+	<div class="pure-menu pure-menu-horizontal">
+	<ul class="pure-menu-list">
+		{{ macros::active_link(expected="index", href="/about", text="About") }}
+		{{ macros::active_link(expected="badges", href="/about/badges", text="Badges") }}
+		{{ macros::active_link(expected="builds", href="/about/builds", text="Builds") }}
+		{{ macros::active_link(expected="metadata", href="/about/metadata", text="Metadata") }}
+		{{ macros::active_link(expected="redirections", href="/about/redirections", text="Redirections") }}
+	</ul>
+	</div>
+</div>
+</div>
+{% endblock %}
```

Comment:
Currently in other templates blocks are indented
```suggestion
{% block header %}
    <div class="cratesfyi-package-container">
    <div class="container about">
        <h3 id="crate-title">Docs.rs documentation</h3>
        <div class="pure-menu pure-menu-horizontal">
            <ul class="pure-menu-list">
                {{ macros::active_link(expected="index", href="/about", text="About") }}
                {{ macros::active_link(expected="badges", href="/about/badges", text="Badges") }}
                {{ macros::active_link(expected="builds", href="/about/builds", text="Builds") }}
                {{ macros::active_link(expected="metadata", href="/about/metadata", text="Metadata") }}
                {{ macros::active_link(expected="redirections", href="/about/redirections", text="Redirections") }}
            </ul>
        </div>
    </div>
    </div>
{% endblock %}
```

Reply:
This makes the lines a lot longer though ... Maybe instead we could change the other templates not to be indented?

The reason I don't think it's very helpful here is because there's only one `{% block %}` usually so it doesn't need to be offset from other similar code.

Reply:
To be honest I still like the indentation more. It does make the lines longer but I think that's a fine price to pay for what's in my opinion better readability and clearer intent